cli/wsproxy.ml

```diff
@@ -13,7 +13,7 @@
  *)
 open Wslib
 
-let get_dir_path () = Printf.sprintf "/var/xapi/" 
+let get_dir_path () = Printf.sprintf "/var/xapi/"
```

Comment:
This could be changed into a variable? :sweat_smile:

Reply:
I'm not even sure it's worth using a variable as it is, seeing that it's not configurable and it's only used in a single line.